src/techCompanies.js

```diff
@@ -1012,6 +1012,13 @@ export const techCompanies = [
         industry: "Tech media/research ",
         website: "https://techcabal.com",
       },
+      {
+        companyName: "Trove",
```

Comment:
Please also include it also in the `README.md` file

Reply:
Have added it @gray-adeyi . You will find the PR as #124

Reply:
Great! I don't have merge access to this repository. You pr will be merged soonest

Reply:
Alright boss, thanks. @gray-adeyi